### PR TITLE
mds: Sanity check if mds_bal_split_bits is over max or not

### PR DESCRIPTION
**Pull request by shinobu-x**
Signed-off-by: Shinobu Kinjo <shinobu@redhat.com>

### CODE REVIEW

**Comment by shinobu-x** [comment]
test this please

File: src/mds/MDBalancer.cc [hunk 1]
```diff
@@ -493,8 +493,14 @@ void MDBalancer::queue_split(const CDir *dir, bool fast)
 
     // Pass on to MDCache: note that the split might still not
     // happen if the checks in MDCache::can_fragment fail.
+    auto mds_bal_split_bits = g_conf->mds_bal_split_bits;
```
Comment (ukernel) [inline src/mds/MDBalancer.cc]: need to make sure "split_dir->get_frag().bits() + mds_bal_split_bits " <= 24

**Comment by shinobu-x** [comment]
we're already doing sanity check in:
https://github.com/ceph/ceph/blob/master/src/common/config.cc#L1099

File: src/mds/MDBalancer.cc (hunk 1 above)
Comment (shinobu-x) [reply src/mds/MDBalancer.cc]: if that's required, this isn't correct is it?
https://github.com/ceph/ceph/blob/master/src/common/options.cc#L6216-L6218

File: src/mds/MDBalancer.cc [hunk 2]
```diff
@@ -493,8 +493,14 @@ void MDBalancer::queue_split(const CDir *dir, bool fast)
...
+    if (mds_bal_split_bits > 24) {
```
Comment (batrick) [inline src/mds/MDBalancer.cc]: Right, so this sanity check is already in `src/common/options.cc`. I don't see the need for this check.

**Pull request closed without merging**